lib/wings/hydra/pcdm/models/concerns/pcdm_valkyrie_behavior.rb

```diff
@@ -0,0 +1,23 @@
+require 'wings/active_fedora_converter'
+require 'wings/valkyrizable'
+
+module Wings
+  module PcdmValkyrieBehavior
+    ##
+    # Gives the subset of #members that are PCDM objects
+    # @return [Enumerable<PCDM::ObjectBehavior>] an enumerable over the members
+    #   that are PCDM objects
+    def objects(valkyrie: false)
+      af_objects = Wings::ActiveFedoraConverter.new(resource: self).convert.objects
+      return af_objects unless valkyrie
+
+      # return Valkyrie Resources
```

Comment:
My only question here is, since `af_objects` are ActiveFedora::Base objects, then they should already be "Valkyrizable" and should respond to `#valkyrie_resource`, is that correct? 

Could it be  that the following block is replaced with `af_objects.map(&:valkyrie_resource)`?

Reply:
The GenericWork class did not respond to #valkyrie_resource which is why I added the include.